templates/collections/blocks/featured_page.html

```diff
@@ -1,3 +1,3 @@
-<ul class="card-group">
+<ul class="row card-group">
```

Comment:
We're mixing BEM and Bootstrap here and should separate them so that the `row` is on a parent - `<div>` perhaps - and the component is within. See: https://github.com/nationalarchives/tna-frontend-design-toolkit/blob/main/GUIDELINES.md#avoid-mixing-bem-and-grid-classes-on-the-same-element